src/plugins/intel_cpu/src/graph_dumper.cpp

```diff
@@ -225,9 +225,11 @@ void serializeToXML(const Graph &graph, const std::string& path) {
     if (path.empty())
         return;
 
+    static int g_idx = 0;
+    std::string xmlPath = std::to_string(g_idx++) + "_" + path;
```

Comment:
what if path is a fullpath like `/home/xxx/dump.xml`, xmlPath would become an invalid path, right?

Reply:
Yes, I will add index as path_**index**.xml.

Reply:
Done